src/Wallabag/CoreBundle/Controller/DeveloperController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Wallabag\CoreBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Wallabag\ApiBundle\Entity\Client;
+use Wallabag\CoreBundle\Form\Type\ClientType;
+
+class DeveloperController extends Controller
+{
+    /**
+     * @Route("/developer", name="developer")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function indexAction()
+    {
+        return $this->render('WallabagCoreBundle:Developer:index.html.twig');
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @Route("/developer/client/create", name="create_client")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function createClientAction(Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $client = new Client();
+        $clientForm = $this->createForm(ClientType::class, $client);
+        $clientForm->handleRequest($request);
+
+        if ($clientForm->isValid()) {
+            $client->setAllowedGrantTypes(array('token', 'authorization_code'));
```

Comment:
I don't know. Before this PR, I followed this gist https://gist.github.com/tjamps/11d617a4b318d65ca583 and there is only password. I need to read some documentation about that. @j0k3r any opinion?